packages/site/src/components/Header.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+import TopLevelNav from './TopLevelNav';
+import { Link as RouterLink } from 'react-router-dom';
+import _Link from '@mineral-ui/link';
+
+type Props = {
+  currentPath?: string
+};
+
+const headerStyles = {
+  display: 'flex',
+  padding: '0 2em',
+  justifyContent: 'space-between'
+};
+
+const Container = createStyledComponent('header', headerStyles);
+const Logo = createStyledComponent(_Link, (props, theme) => ({
+  // TODO figure out glamorous media queries for mobile
+  display: 'inline-flex',
+  alignItems: 'center',
+  height: 100,
+  textDecoration: 'none',
+  letterSpacing: '-0.2em',
+
+  '& > :first-child': {
+    display: 'block',
+    marginRight: 20,
+    borderRadius: 40,
+    width: 80,
+    height: 80,
+    backgroundImage:
+      'url(http://www.treasuremountainmining.com/image/Wulfenite.jpg)',
```

Comment:
Please do not link to an image on someone else's site, unless it is a site designed for that, or they give you permission.